samples/CarterSample/Features/Actors/Actor.cs

```diff
@@ -1,9 +1,14 @@
 namespace CarterSample.Features.Actors
 {
+    using System.Text.Json.Serialization;
```

Comment:
Should this be a carter-specific attribute instead? Given Carter can be configured with it's JSON Serialiser, it's possible to mix-and-match with it here. If using Newtonsoft this seems a bit odd to me